modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/SourceEditor.soy

```diff
@@ -64,11 +64,10 @@
 				</svg>
 			</span>
 
-			{call liferay.frontend.Tooltip.render}
+			{call ClayTooltip.render}
 				{param elementClasses: 'source-editor__fixed-text__help__tooltip' /}
-				{param position: 3 /}
 				{param selector: '.source-editor__fixed-text__help' /}
-				{param title: $helpMessage /}
+				{param _content: $helpMessage /}
```

Comment:
ClayTooltip component was not designed to be used this way; `_content` is internal mechanism of the component, indicated by `_` prefix and `.internal()`. Unfortunately, there is no existing use in portal. I would assume you only need to set `title` attribute of your target. If it doesn't work, please contact the person who make this component and ask what is the intended use.